server/filters/formatGoalStatusValue.ts

```diff
@@ -3,7 +3,7 @@ export default function formatGoalStatusValue(value: string): string {
   return statusValue
 }
 
-export enum GoalStatusValue {
+enum GoalStatusValue {
```

Comment:
These didnt need to be exported, and are effectively private to the internals of the filter function 👍